src/Billing.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Billing endpoint helper methods.
+ *
+ * @package Pinterest_For_WooCommerce/Classes
+ */
+
+namespace Automattic\WooCommerce\Pinterest;
+
+use Automattic\WooCommerce\Pinterest\API\Base;
+use Automattic\WooCommerce\Pinterest\Logger;
```

Comment:
This import is unnecessary, since the namespace for this class is the same as `Logger`.